langchain/vectorstores/chroma.py

```diff
@@ -192,31 +198,6 @@ def similarity_search_by_vector(
         )
         return _results_to_docs(results)
 
-    def similarity_search_by_vector_with_relevance_scores(
```

Comment:
why did you remove this method? this is why test is failing.

Reply:
> why did you remove this method? this is why test is failing.

@rlancemartin I am not sure. Totally unintended. Thanks for pointing it out.